src/core/symbology-ng/qgsgraduatedsymbolrendererv2.h

```diff
@@ -139,8 +140,8 @@ class CORE_EXPORT QgsGraduatedSymbolRendererV2 : public QgsFeatureRendererV2
 
     virtual QgsSymbolV2List symbols() override;
 
-    QString classAttribute() const { return mAttrName; }
-    void setClassAttribute( QString attr ) { mAttrName = attr; }
+    QString classAttribute() const { return mExpression.data() ? mExpression->dump() : ""; }
```

Comment:
It's better to use QString() rather than "" to avoid creation of temporary strings